docs-website/_docs/config.md

```diff
@@ -14,20 +14,42 @@ The structure of the config file can be either `json` or `yaml` and is passed us
 ```json
 {
   "severity_overrides": {
-    "CUS002": "ERROR",
+    "CUSTOM001": "ERROR",
```

Comment:
the code for custom checks is arbitrary but the choice to document and guide towards `CUS0XX` is deliberate to maintain consistency across the custom and builtin checks when reviewing output reports. I don't think anything is gained by updating this to CUSTOM